templates/sds-bamm-macros.tpl

```diff
@@ -176,8 +174,8 @@ a bamm-c:RangeConstraint ;
 {{ member.name }}
 {%- elif member.datatype and member.datatype.endswith("_t") -%}
 Characteristic{{ snake_to_camel(member.datatype.capitalize()) }}
-{%- elif member.type and member.type.endswith("_t") -%}
-Characteristic{{ snake_to_camel(member.type.capitalize()) }}
+{%- elif member.datatype and member.datatype.endswith("_t") -%}
```

Comment:
The lines 177-178 can actually be removed now when it is unified that "datatype" always is "datatype" (previously it was called "type" for attributes") so the workaround here can be removed

Reply:
Ok. Will do. Thanks for the hint.